### PR TITLE
[TRAFODION-1936] Update copyright banners for 2016

### PR DESCRIPTION
**Pull request by svarnau**
Also removed a redundant license header from one source file.

### CODE REVIEW

**Comment by robertamarton** [comment]
Looks good to me

**Comment by svarnau** [comment]
jenkins, retest please -- hbase/zookeeper issues in previous test run unrelated to this change.

File: win-odbc64/odbcclient/Drvr35Res/Drvr35Res.rc [hunk 1]
```diff
@@ -93,7 +93,7 @@ BEGIN
             VALUE "FileDescription", "TRAF ODBC Driver Resource DLL"
             VALUE "FileVersion", "1.3.0.0"
             VALUE "InternalName", "TRAF ODBC Driver Resource DLL"
-            VALUE "LegalCopyright", "?Copyright 2015 Apache Trafodion"
```
Comment (DaveBirdsall) [inline win-odbc64/odbcclient/Drvr35Res/Drvr35Res.rc]: Why does the copyright say, "Apache Trafodion" instead of, say, "Apache Software Foundation"?

**Comment by DaveBirdsall** [comment]
Why do the copyrights say "Apache Trafodion" instead of "Apache Software Foundation"?

**Comment by svarnau** [comment]
That is a heck of a good question. I was only paying attention to updating the year, but I think you are correct. It should match the copyright that is in the NOTICE file -- ASF.

**Pull request merged**